### PR TITLE
Fix bug, adding missing quotes.

### PR DESCRIPTION
**Pull request by ashercodeorg**
image

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
Do we know when/why this broke? Seems like this code hasn't changed since september https://github.com/code-dot-org/code-dot-org/blame/staging/dashboard/public/frequency/frequency.js#L1156.

Change looks fine to me if it fixes things. cc @Hamms who appears to be responsible for all the commits in this file.

**Comment by ashercodeorg** [comment]
The widget appears functional, even with the JS error. The JS error was found as a result of the `Extra Links` menu not opening and closing.

**Comment by ashercodeorg** [comment]
And I have no idea if it fixes things, I'm just under the impression that href links should be quoted.

File: dashboard/public/frequency/frequency.js [hunk 1]
```diff
@@ -1153,7 +1153,7 @@ $(document).ready(function () {
   $("#sort-toggle button").click(bg.handleSortChange.bind(bg));
 
   // When we switch back to shift mode, force an alphabetic order
-  $("a[href=#shift]").click(function () {
+  $("a[href='#shift']").click(function () {
     $("#sort-toggle button[value=alphabetic]").trigger('click');
```
Comment (Hamms) [inline dashboard/public/frequency/frequency.js]: this should probably be quoted, too; `$("#sort-toggle button[value='alphabetic']")`

**Comment by Hamms** [comment]
Nice catch! Thanks for fixing. LGTM modulo one comment

File: dashboard/public/frequency/frequency.js (hunk 1 above)
Comment (ashercodeorg) [reply dashboard/public/frequency/frequency.js]: Done.

**Pull request merged**